src/screens/Notifications/ConfirmEmailScreen.tsx

```diff
@@ -192,26 +173,27 @@ class ConfirmEmailScreen extends Component<Props, State> {
           </Text>
         </View>
         <View style={styles.inputItemContainer}>
-          <CodeInput value={this.state.code} onTextChange={this.onChange} isError={!!this.state.error} />
-          <Text style={styles.error}>{this.state.error}</Text>
+          <CodeInput value={this.state.code} onTextChange={this.onChange} isError={!!notificationError} />
```

Comment:
double negation here is not necessary. Empty string is still falsy, so without negation, effect will be just the same

Reply:
it is needed, the isError is a boolean type, not a string

Reply:
ah, OK